### PR TITLE
Feature: README marketing blurb

### PR DESCRIPTION
**Pull request by fmitchell-r7**
This adds a "why", "what", "who" marketing blurb to the top of the README. In addition, there's a tagline that should eventually go in the GitHub repository description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+We built propsd with lessons learned from years of running [Conqueso][] on
+large scale systems. High availability is achieved by leveraging [Amazon S3][]
+to deliver properties and [Consul][] to handle service discovery. Composable
+layering lets you set properties for an organization, a single server, and
+everything in between. Plus, flat file storage makes backups and audits
+a breeze.
+
+So if your [Conqueso][] server's starting to heat up, you just want an audit
+trail when things change, give propsd a try.
```
Comment (dgreene-r7) [inline README.md]: This parses funny. I think you want:
> So if your Conqueso server is starting to heat up or you just want an audit trail when things change then give propsd a try.

Another option is:
>So if your Conqueso server is starting to heat up, you just want an audit trail when things change, or \<INSERT ANOTHER REASON HERE\>, give propsd a try.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Propsd
 [![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url]
+
+Propsd does dynamic property management at scale, from thousands of servers to
+hundreds of developers.
```
Comment (dgreene-r7) [inline README.md]: What are you trying to say here? Maybe:
> Propsd does dynamic property management at scale, across thousands of servers and contributions from hundreds of developers.

**Comment by dgreene-r7** [comment]
:+1:

**Comment by athompson-r7** [comment]
This is great :+1:

**Pull request merged**